install-from-binstall-release.sh

```diff
@@ -27,8 +27,11 @@ fi
 
 ./cargo-binstall -y --force cargo-binstall
 
-if ! [[ ":$PATH:" == *":$HOME/.cargo/bin:"* ]]; then
+if ! [[ $CARGO_HOME ]]; then
+    CARGO_HOME=$HOME/.cargo
```

Comment:
Actually maybe quoting these would be prudent, in case someone has a space in their HOME

Reply:
This is not needed when doing assignments:

```bash
foo="a b c"
bar=$foo/123
echo "$bar"
```

outputs `a b c/123`

Reply:
I've verified that it indeed isn't necessary, closing this conversion as resolved.